pacman/utilities/json_utils.py

```diff
@@ -322,6 +322,7 @@ def graph_to_json(graph):
 
 def graph_from_json(json_dict):
     json_dict = json_to_object(json_dict)
+    # TODO App level if exists needed
```

Comment:
new TODO. was that intentional. or were you going to do something and forgot?

Reply:
Intentional todo to makr out of scope area.